python/foglamp/tasks/north/sending_process.py

```diff
@@ -303,67 +220,60 @@ class SendingProcess:
             "type": "integer",
             "default": "60"
         },
-        "sleepInterval": {
-            "description": "Time in seconds to wait between duration checks",
-            "type": "integer",
-            "default": "1"
-        },
         "source": {
-            "description": "Source of data to be sent on the stream. "
-                           "May be either readings, statistics or audit.",
+            "description": "Source of data to be sent on the stream. May be either readings, statistics or audit.",
             "type": "string",
-            "default": _DATA_SOURCE_READINGS
+            "default": "readings"
         },
         "blockSize": {
             "description": "Bytes to send in each transmission",
             "type": "integer",
             "default": "500"
         },
+        "sleepInterval": {
+            "description": "Time in seconds to wait between duration checks",
+            "type": "integer",
+            "default": "1"
+        },
+        'plugin': {
+            'description': 'The name of the translator to use to translate the readings into the output format and send them.',
+            'type': 'string',
+            'default': 'omf'
+        },
+        'filter': {
+            'description': 'Data filtering rules to apply to the data that has been pulled from the storage layer.',
+            'type': 'string',
+            'default': 'None'
```

Comment:
I think it should be applied with keys! `applyFilter`? if yes then respect `filterRule` - which has default value `.[]`